simtools/visualization/legend_handlers.py

```diff
@@ -176,12 +166,7 @@ class HexOffPixelHandler:
     """
 
     @staticmethod
-    def legend_artist(*args, **kwargs):
-        handlebox = kwargs.pop("handlebox", None)
-        if handlebox is None:
-            # handlebox is expected, return None if not provided
-            return None
-
+    def legend_artist(_, __, ___, handlebox):
         x0, y0 = (
             handlebox.xdescent + handlebox.width / 3,
```

Comment:
This line is repeated quite frequently. Is there an easy way to avoid it?

Reply:
I have added a function to return the center values.